.github/workflows/test_iree.yml

```diff
@@ -162,57 +164,37 @@ jobs:
             --timeout=1200 \
             --durations=0 \
             --config-files=${MODELS_CONFIG_FILE_PATH}
-
-      - name: "Running real weights SDXL prompt encoder tests"
-        id: prompt_encoder
+      
+      - name: "Running SDXL special model tests"
+        id: special_models_sdxl
         if: ${{ !cancelled() }}
         run: |
           source ${VENV_DIR}/bin/activate
-          pytest iree_tests/pytorch/models/sdxl-prompt-encoder-tank \
+          pytest iree_special_models/sdxl \
```

Comment:
Assuming that the new directories don't end up nested under `iree_tests`, they should be added to the 'paths' that trigger this workflow https://github.com/nod-ai/SHARK-TestSuite/blob/4751fab06fdc08818ee30530e0cdfc38fc6ce4b7/.github/workflows/test_iree.yml#L7-L16